cmd/cli/install.go

```diff
@@ -350,6 +356,12 @@ func (i *installCmd) validateOptions() error {
 		}
 	}
 
+	if i.deployPrometheus {
+		if !i.enablePrometheusScraping {
+			return errors.Errorf("Prometheus scraping cannot be disabled when deploying Prometheus.")
```

Comment:
Nice!

Reply:
What is the reason to add this check? I can imagine a use case to deploy prometheus, but only turn on scraping much later on via the ConfigMap.

Reply:
That's fair. Mostly I wanted to protect the customer from doing something they may not have been aware of. This can be worked around by creating the deployment and then turning it off in the config map. Or we can just remove this check. Depends on what we think is best for the user.

Reply:
I'd suggest we remove the error, and print a message stating `prometheus_scraping` needs to be enabled for metrics to actually be scraped. Deployment of prometheus should be decoupled from scraping configuration.